src/core/Object3D.js

```diff
@@ -662,6 +668,20 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 		}
 
+		// SkinnedMesh specific
+
+		if ( this.skeleton !== undefined ) {
```

Comment:
Hi

Why not move this to a toJSON inside SkinnedMesh i think it would make more sense to move mesh serialization specifics to the Mesh and SkinnedMesh objects!

Thanks

Reply:
Thinking to restructure `Object3D.toJSON()` before merging this PR...